modules/internal/ChapelIO.chpl

```diff
@@ -426,18 +783,42 @@ module ChapelIO {
   // these are overridden to not be inout
   // since they don't change when read anyway
   // and it's much more convenient to be able to do e.g.
-  //   reader & new ioLiteral("=")
+  //   reader <~> new ioLiteral("=")
+
+  /* Overlead to support reading an ioLiteral without
```

Comment:
Also, please make ioLiteral a link to the structure, in case readers reached this function through a different path than the top of the documentation for this module.